fuse/archivemount/Portfile

```diff
@@ -1,30 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           fuse 1.0
-
-name                archivemount
-version             0.9.1
-revision            1
-categories          fuse
-platforms           darwin
-license             GPL-2
-maintainers         nomaintainer
-
-description         File system for accessing archives using libarchive
-long_description \
-    Archivemount is a piece of glue code between libarchive and FUSE \
-    It can be used to mount a (possibly compressed) archive \
-    (as in .tar.gz or .tar.bz2) and use it like an ordinary filesystem.
-
-homepage            https://www.cybernoia.de/software/archivemount.html
-master_sites        https://www.cybernoia.de/software/archivemount/
-
-checksums           rmd160  332c5641072eb76f7aabae2b829254591c21a060 \
-                    sha256  c529b981cacb19541b48ddafdafb2ede47a40fcaf16c677c1e2cd198b159c5b3 \
-                    size    133251
-
-patchfiles          patch-archivemount.diff
-
-depends_lib-append \
-    port:libarchive
+PortSystem              1.0
+PortGroup               fuse 1.0
+PortGroup               sourcehut 1.0
+PortGroup               legacysupport 1.1
+PortGroup               makefile 1.0
+
+# Epoch indicates new upstream, must now stay
+# See: https://trac.macports.org/ticket/70623
+sourcehut.setup         nabijaczleweli archivemount-ng 1
+name                    archivemount
+revision                0
+epoch                   1
+categories              fuse
+license                 GPL-2 BSD
+maintainers             {@TheRealKeto gmail.com:therealketo} openmaintainer
+
+description             File system for accessing archives using libarchive
+long_description        Archivemount is a piece of glue code between libarchive \
+                        and FUSE. It can be used to mount a (possibly compressed) \
+                        archive (as in .tar.gz or .tar.bz2) and use it like an \
+                        ordinary filesystem.
+
+checksums               rmd160  614826c4fca3cddd24797dff9232a34b78d252ef \
+                        sha256  907608dfea503088194d2244443873c7b5293f294280348e9873ff22054fd7e4 \
+                        size    29066
+
+# Already on upstream, but no new release made
+patch.pre_args-replace  -p0 -p1
+patchfiles-append       apple-standard-st_mtim-st_mtimespec.patch
+
+depends_lib-append      port:libarchive
+
+compiler.cxx_standard   2020
+
+platform darwin powerpc {
```

Comment:
This in fact has nothing to do with powerpc and should be applied unconditionally. (If you search through MacPorts directory, there are quite a number of similar patches. Many of those have been upstreamed, see `icu`, for example, or `libjxl`.)

Reply:
@TheRealKeto For example: https://github.com/unicode-org/icu/pull/2250/commits/c9e1c1343a7b25510462afea36c83260ee6a6da9

Reply:
Sounds good. I've renamed the patch to reflect its actual function and will apply it unconditionally in latest commit; I might try to get this to upstream as well.